lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -220,7 +220,7 @@ def define_local_variables
       @machine_extra_param = []
       @device_extra_param = []
       @iommu_device_param = []
-      @cpu_options = []
+      @cpu_options = ['@cpu@', '+x2apic', '+fsgsbase', 'model=@cpu_model@']
```

Comment:
I think it's better not to put `@cpu@` here; `@cpu@` is different from other options in terms that it must be present before the other options.

Also, here is some Ruby magic:
```suggestion
      @cpu_options = %w[@cpu@ +x2apic +fsgsbase model=@cpu_model@]
```